frontend/.env

```diff
@@ -6,6 +6,7 @@ REACT_APP_BASE_URL=https://simplereport.gov/
 REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b
 REACT_APP_IS_TRAINING_SITE=false
 INLINE_RUNTIME_CHUNK=false
+REACT_APP_CONTENT_SECURITY_POLICY=default-src 'self';script-src 'self' dap.digitalgov.gov www.google-analytics.com;style-src 'self' 'unsafe-inline';img-src 'self' data:;connect-src 'self'  https://us-street.api.smartystreets.com www.google-analytics.com https://dc.services.visualstudio.com/v2/track %REACT_APP_BACKEND_URL%;
```

Comment:
Can an entry in this file interpolate another entry?

(Also, is `%REACT_APP_BACKEND_URL%` different from `self` in any environment?)

Reply:
Good catch

that won't work